app/api-v1/services/recipeService.js

```diff
@@ -1 +1,15 @@
-module.exports = {}
+const { addRecipe } = require('../../db')
+
+async function createRecipe(reqBody) {
+  if (!reqBody) {
+    return null
+  }
+
+  const recipe = await addRecipe(reqBody)
+
+  return recipe
```

Comment:
You'll need to return a status code here as well as the result as an object:
{ statusCode: 201, result }

Reply:
I've taken a controversial approach of putting the http specific functionality in the `routes` directory, so this remains business logic only. what do you think about that separation of concerns?

Reply:
I would not call it a service, maybe creating a `db/models/` folder and adding it there. Personally I like to keep controllers in one folder excluding any helpers or nesting e.g.

```
api/v1/controller1/index.js
api/v1/controller1/utils/helpers.js
api/v1/controller1/Recipe.js
```

easier to read, also since it's performing some operations and might need to call oother method outside of this entity we culd put login in  `db/models/Recipe.js`